L/libflame/build_tarballs.jl

```diff
@@ -0,0 +1,89 @@
+]
+
+# Bash recipe for building across all *64-bit* platforms
+# We try to maintain consistency with the blis Yggdrasil build scripts.
+
+script = raw"""
+cd $WORKSPACE/srcdir/libflame
+
+# We might need newer `config.guess`` and `config.sub` files
+update_configure_scripts
+
+extra_flags=""
+
+if [[ "${target}" == *-apple-* ]]; then
+    atomic_patch -p1 ${WORKSPACE}/srcdir/patches/mac-update-makefile.patch
+fi
+
+if [[ "${target}" == *-w64-mingw32* ]]; then
+    extra_flags+=" --enable-windows-build "
+    # disable time & clock functions on windows (mingw): sys/times.h is missing
+    atomic_patch -p1 ${WORKSPACE}/srcdir/patches/windows-remove-time.patch
+    # update the make build scripts for windows cross-compiling with gcc:
+    atomic_patch -p1 ${WORKSPACE}/srcdir/patches/windows-update-build.patch
+fi
+
+if [[ "${target}" == *mingw* ]]; then
+    LBT="-L${libdir} -lblastrampoline-5"
+else
+    LBT="-L${libdir} -lblastrampoline"
+fi
+
+# - Compile and build a LAPACK compatibility layer with --enable-lapack2flame
+# - If a static library is not needed, use --disable-static-build
+# - Enable a dynamic build with --enable-dynamic-build
+./configure \
+    --prefix=${prefix} --build=${MACHTYPE} --host=${target} \
+    --enable-multithreading=openmp \
+    --enable-lapack2flame \
+    --disable-static-build \
+    --enable-dynamic-build \
+    --enable-max-arg-list-hack \
+    --disable-autodetect-f77-ldflags --disable-autodetect-f77-name-mangling \
+    $extra_flags
+
+make -j${nproc} BLAS_LIB="${LBT}"
+make install
+
+install_license LICENSE
+"""
+
+# These are the platforms we will build for by default, unless further
+# platforms are passed in on the command line
+platforms = supported_platforms()
+
+filter!(p -> arch(p) != "i686", platforms)
+filter!(p -> arch(p) != "armv7l", platforms)
+filter!(p -> arch(p) != "armv6l", platforms)
+filter!(p -> arch(p) != "powerpc64le", platforms)
+
+# Enable the following line if trying to link against Fortran code:
+# platforms = expand_gfortran_versions(platforms)
+
+# Dependencies that must be installed before this package can be built
+dependencies = [
+    Dependency(PackageSpec(name="libblastrampoline_jll", uuid="8e850b90-86db-534c-a0d3-1478176c7d93"), compat="5.9.0"),
+    # For OpenMP we use libomp from `LLVMOpenMP_jll` where we use LLVM as compiler (BSD
+    # systems), and libgomp from `CompilerSupportLibraries_jll` everywhere else.
+    Dependency(PackageSpec(name="CompilerSupportLibraries_jll", uuid="e66e0078-7015-5450-92f7-15fbd957f2ae"); platforms=filter(!Sys.isbsd, platforms)),
+    Dependency(PackageSpec(name="LLVMOpenMP_jll", uuid="1d63c593-3942-5779-bab2-d838dc0a180e"); platforms=filter(Sys.isbsd, platforms)),
+]
+
+products = [
+    LibraryProduct("libflame", :libflame),
+]
+
+# Build the tarballs
+build_tarballs(ARGS, name, version, sources, script, platforms, products, dependencies;
+               preferred_gcc_version=v"11", lock_microarchitecture=false, julia_compat="1.6")
```

Comment:
```suggestion
               preferred_gcc_version=v"11", clang_use_lld=false, lock_microarchitecture=false, julia_compat="1.6")
```
Lets try not using LLD and instead using the other linker to see if these flags work then.

Reply:
Mixed up the gcc ld `-whole-archive` vs. clang lld `-force_load` but didn't seem to like it anyway. Might need to try your approach.

Reply:
> Mixed up the gcc ld `-whole-archive` vs. clang lld `-force_load`

1. this isn't gcc vs clang, but Linux vs macOS
1. this is why we provide the [`flagon`](https://docs.binarybuilder.org/stable/build_tips/#utils_build_env) utility.

Reply:
Haven't looked at the log, but is this a case of #10290?

Reply:
What were the symptoms of that error with single module?